src/editor.c

```diff
@@ -2391,6 +2391,8 @@ typedef struct
 
 
 #define CURSOR_PLACEHOLDER "_" /* Would rather use … but not all docs are unicode */
+
+
```

Comment:
I didn't want to make a whole new commit just to add a couple newlines, so I put it here.

It might be nice later to move this macro and the structure definition that precedes it to the top of the file where it is customary.